question11/main.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+// loading img 
+const wrapper = document.getElementById('wrapper');
+const loadImg = document.createElement('img');
+const showLoadImg = () => {
+    loadImg.src = "loading-circle.gif";
+    wrapper.appendChild(loadImg);
+} 
+
+async function callApi() {
+    showLoadImg();
+    try {
+        const res = await fetch('https://jsondata.okiba.me/v1/json/dFQLo210903231659');
+        const json = await res.json();
+        return json.data;
+    }
+    catch(error) {
+        wrapper.textContent = "データが取得できませんでした";
+        console.error(error);
+    }
+    finally{
+        loadImg.remove()
+    }}
```

Comment:
In the current code, createList(); will be executed even if no data is sent due to some error.
I think it would be better to change the code so that the message "Data could not be retrieved" is displayed when an error occurs.

IMO
I left a comment out in Japanese.（I'm sorry, I don't have the ability to speak in English😭）
[codesandbox](https://codesandbox.io/s/detagaqu-de-dekimasendesitagabiao-shi-saretehosiidesu-fqi5x?file=/main.js) If you'd like to check it out, please do.

Reply:
Thank you for the your review and kindness!!!! I think I understood what you meant. I fixed the code as below.

When error is occur, 
①null is returned
②If it is NOT null, run createList().

I do not have a confidence..This is just a idea so I would like to hear your opinion.
https://github.com/csakyo/homework/pull/23/commits/22c4624fecdad192127ecc6ee708d3b017040189

Reply:
It's looking good!